sqlair.go

```diff
@@ -92,15 +100,66 @@ type Iterator struct {
 	started bool
 }
 
+// cache is a cache for statements prepared on a database/transaction.
+type cache struct {
```

Comment:
Given the earlier points we might not need this anymore, but if we keep it for some reason let's please give it a more distinctive name.

Reply:
Done.